src/application.rs

```diff
@@ -209,6 +220,16 @@ impl SwayOSDApplication {
 						}
 					}
 				}
+				"max-volume" => {
+					let value = child.value().str().unwrap_or("").trim();
+					match value.parse::<u8>() {
+						Ok(_) => (OsdTypes::MaxVolume, Some(value.to_string())),
+						Err(_) => {
+							eprintln!("{} is not a number between 0 and {}!", value, u8::MAX);
+							return 1;
+						}
+					}
+				}
```

Comment:
I'd rather set the max-volume when running `--output-volume=raise`. So a loop here checking multiple args + making sure that the max-volume always gets set before running `--output-volume=raise`

Reply:
Do you mean that I should remove `--max-volume` and add another argument like so `--output-volume=raise [MAX_VOLUME]`?

Reply:
I meant being able to run it like this `swayosd --max-volume=150 --output-volume=raise` instead of needing to run `swayosd --max-volume=150` once after starting swayosd

Reply:
Also now instead of still calling activate_action if the Argument is from type None, it just does not do anything and immediately exits

Reply:
Looks good! We should reset the max-volume after each call to the default value (100)

Reply:
You mean after someone uses --max-volume=150 we should reset it to 100 after the next time the volume changes? That seems like a bad idea to me. Starting SwayOSD and then with 1 command always setting it to 150 until you shut down your PC seems more sensible to me. This also does not disable your ability to always use max-volume when raising volume meaning it increases versatility.

The bug though I will immediately fix

Reply:
> Starting SwayOSD and then with 1 command always setting it to 150 until you shut down your PC seems more sensible to me

I can agree with that. In that case we should be able to start swayosd with `--max-volume=150`, so `swayosd --max-volume=150`. Seems kinda janky if the user needs to run two commands at start. Any input/suggestions for a alternative? :)

Reply:
It is done, now any command can be executed on start as that was the simplest way to do it.